src/lib/y2partitioner/actions/edit_blk_device.rb

```diff
@@ -92,12 +92,17 @@ def title
         end
       end
 
-      # Extended partitions and LVM thin pools cannot be edited
+      # Checks whether it is possible to edit the device
       #
-      # @note An error popup is shown when the edit action cannot be performed.
+      # It also warns the user when the action is possible but it does not make sense, like editing
```

Comment:
I wouldn't say that editing a snapshots makes no sense. It's true that formatting it would be a bit crazy, but mounting it could be useful in some scenarios (actions for both formatting and mounting are in the "edit" wizard).